src/graphql/main/schema.graphql

```diff
@@ -846,21 +853,28 @@ input OnChainUsdPaymentSendAsBtcDenominatedInput {
   address: OnChainAddress!
   amount: SatAmount!
   memo: Memo
-  targetConfirmations: TargetConfirmations = 1
+  requestId: PayoutRequestId
+  speed: PayoutSpeed = FAST
+  targetConfirmations: TargetConfirmations = 0
+    @deprecated(reason: "Ignored - will be replaced")
   walletId: WalletId!
 }
 
 input OnChainUsdPaymentSendInput {
   address: OnChainAddress!
   amount: CentAmount!
   memo: Memo
-  targetConfirmations: TargetConfirmations = 1
+  requestId: PayoutRequestId
```

Comment:
I think this idempotency key should be at the HEADER level.

Reply:
From ACE, we decided that this would be handled more generically via middleware potentially later on, and for now we can remove from the schema and not enforce idempotency at that level (as is the current state of the code)